### ISSUE TITLE
Error displaying an empty Storage Access Policy

### ISSUE THREAD

**Issue by ishepherd** [microsoft/AzureStorageExplorer#4196]
**Storage Explorer Version**: 1.18.0
**Build Number**: 20210227.4
**Platform/OS**: Windows 10
**Architecture**: x64
**Regression From**:  Unknown

## Steps to Reproduce ##
1. Create a storage account
2. Create a stored access policy, but forget to add any permissions or date restrictions:
  `az storage container policy create --account-name $storageAccount --container-name $BlobContainer --name $BlobContainer-BackupsPolicy`
3. In Azure Storage Explorer, right-click the blob container and select "Manage Access Policies..."

## Expected Experience ##

I would see an empty storage access policy - you can see what Portal shows for this case.

## Actual Experience ##

* Error in Activities log: "Failed to get access policies for blob container '<name>'".
* "Details..." output:
```
{
  "name": "TypeError",
  "message": "Cannot read property 'startsOn' of undefined",
  "stack": "TypeError: Cannot read property 'startsOn' of undefined\n    at C:\\Users\\livec\\AppData\\Local\\Programs\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-blob-extension\\dist\\src\\Actions\\BlobContainerActions.js:754:58\n    at Array.map (<anonymous>)\n    at Object.<anonymous> (C:\\Users\\livec\\AppData\\Local\\Programs\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-blob-extension\\dist\\src\\Actions\\BlobContainerActions.js:750:75)\n    at step (C:\\Users\\livec\\AppData\\Local\\Programs\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-blob-extension\\node_modules\\tslib\\tslib.js:143:27)\n    at Object.next (C:\\Users\\livec\\AppData\\Local\\Programs\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-blob-extension\\node_modules\\tslib\\tslib.js:124:57)\n    at fulfilled (C:\\Users\\livec\\AppData\\Local\\Programs\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-blob-extension\\node_modules\\tslib\\tslib.js:114:62)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)"
}
```

## Additional Context ##
* I would think there is not much legitimate use for a stored access policy with literally no permissions.
    * Nonetheless it appears that Azure CLI, Azure Portal, and Azure Storage support it fine
    * (Azure Portal lets you edit it to add conditions later).

* I get the impression that Stored Access Policies are not widely used... I also discovered https://github.com/Azure/azure-cli/issues/17278 during this task.

**Comment by MRayermannMSFT** [comment]
Thanks @isheperd for reporting. It seems the [typings](https://github.com/Azure/azure-sdk-for-js/issues/14293) in the new blob SDK led us astray. We'll fix this for our next version. Until then, you can (as you said) clear the policy from the Azure portal (whenever today's Azure AD outage is over that is).

**Comment by MRayermannMSFT** [comment]
I checked in a fix for this that works around the SDK issue. Hopefully they get the typings fixed on their side eventually and I can remove said workaround. Anyway, workaround will be shipped in 1.19
